packages/vite/src/node/ssr/ssrManifestPlugin.ts

```diff
@@ -40,6 +44,50 @@ export function ssrManifestPlugin(config: ResolvedConfig): Plugin {
               })
             }
           }
+          if (chunk.code.includes(preloadMethod)) {
+            // generate css deps map
+            const code = chunk.code
+            let imports: ImportSpecifier[]
+            try {
+              imports = parseImports(code)[0].filter((i) => i.d > -1)
+            } catch (e: any) {
+              this.error(e, e.idx)
+            }
+            if (imports.length) {
+              for (let index = 0; index < imports.length; index++) {
+                const { s: start, e: end, n: name } = imports[index]
+                // check the chunk being imported
+                const url = code.slice(start, end)
+                const deps: string[] = []
+                if (isQuoted(url)) {
+                  const ownerFilename = chunk.fileName
+                  // literal import - trace direct imports and add to deps
+                  const analyzed: Set<string> = new Set<string>()
+                  const addDeps = (filename: string) => {
+                    if (filename === ownerFilename) return
+                    if (analyzed.has(filename)) return
+                    analyzed.add(filename)
+                    const chunk = bundle[filename] as OutputChunk | undefined
+                    if (chunk) {
+                      const cssFiles = chunkToEmittedCssFileMap.get(chunk)
+                      if (cssFiles) {
+                        cssFiles.forEach((file) => {
+                          deps.push(`/${file}`)
+                        })
+                      }
+                      chunk.imports.forEach(addDeps)
+                    }
+                  }
+                  const normalizedFile = path.posix.join(
```

Comment:
Probably can use the existing `normalizePath` util

Reply:
There are two function named `normalizePath` from `@rollup/pluginutils` and `vite utils`, i can't determine whether they have the same feature